lib/deploy/s3/index.js

```diff
@@ -24,11 +24,12 @@ const client = s3.createClient({
 
 const deploy = (dir, {
   bucket,
-  prefix
+  prefix,
+  sync
 }) => {
   const params = {
     localDir: dir,
-    deleteRemoved: true,
+    deleteRemoved: sync,
```

Comment:
`deleteRemoved` is closer to the level of danger this option represents. It's pretty obvious what it'll do.